kubernetes/Chart.yaml

```diff
@@ -4,15 +4,9 @@ description: OISP Helm Chart
 name: oisp
 version: 0.1.0
 dependencies:
-  - name: keycloak
-    version: 6.0.4
-    repository: https://codecentric.github.io/helm-charts
-  - name: kafka
-    version: 0.20.8
-    repository: "@incubator"
-    condition: production
-  - name: kafka
```

Comment:
Why is the kafkatest instance removed?

Reply:
Not intentionally, I is added back on the final commit.